docs/plugins/outputs/timber.asciidoc

```diff
@@ -0,0 +1,229 @@
+:plugin: timber
```

Comment:
@jsvd Looks like this file is new? If so, before you run the build, you should add it to overview table and the list of includes in outputs.asciidoc: https://github.com/elastic/logstash-docs/blob/6.0/docs/plugins/outputs.asciidoc

Otherwise, the topic won't build.